internal/errs/unwrap.go

```diff
@@ -11,7 +11,7 @@ func UnwrapExitCode(err error) int {
 		return 0
 	}
 
-	var eerr interface{ ExitCode() int }
+	var eerr ExitCodeable
 	isExitError := errors.As(err, &eerr)
 	if isExitError {
```

Comment:
nit suggestion ninja refactor:
```suggestion
	if errors.As(err, &eerr) {
```